android/allegro_activity/src/Clipboard.java

```diff
@@ -0,0 +1,88 @@
+package org.liballeg.android;
```

Comment:
I forgot to grep the android directory for tabs :p.  I'll fix it and make it all spaces.